### PR TITLE
:op-guide: Update TiDB flag docs

### PR DESCRIPTION
**Pull request by shenli**
@siddontang @qiuyesuifeng @queenypingcap PTAL

### CODE REVIEW

**Comment by shenli** [comment]
@coocood PTAL

**Comment by ngaut** [comment]
PTAL @queenypingcap

File: op-guide/configuration.md [hunk 1]
```diff
@@ -30,7 +35,9 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 ### --status
 + the status report port for TiDB server
 + default: "10080"
-+ This is used to get server status.
++ This is used to get server internal data. The data includes [prometheus metrics](https://golang.org/pkg/net/http/pprof/) and [pprof](https://golang.org/pkg/net/http/pprof/).
++ Prometheus metrics can be get through "http://host:status_port/metrics".
```
Comment (queenypingcap) [inline op-guide/configuration.md]: get->got

File: op-guide/configuration.md [hunk 2]
```diff
@@ -30,7 +35,9 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
...
++ Pprof data can be get through "http://host:status_port/debug/pprof".
```
Comment (queenypingcap) [inline op-guide/configuration.md]: get -> got

File: op-guide/configuration.md [hunk 3]
```diff
@@ -43,13 +50,14 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 + You can use the "/tmp/tidb.sock" file.
 
 ### --perfschema
-+ enable(1) or disable(0) the performance schema
-+ default: "0"
-+ the value can be (1) or (0). (1) is to enable and (0) is to disable. The Performance Schema provides a way to inspect internal execution of the server at runtime. See [performance schema](http://dev.mysql.com/doc/refman/5.7/en/performance-schema.html) for more information. If you enable the performance schema, the performance will be affected.
++ enable(true) or disable(false) the performance schema
++ default: false
++ the value can be (true) or (false). (true) is to enable and (false) is to disable. The Performance Schema provides a way to inspect internal execution of the server at runtime. See [performance schema](http://dev.mysql.com/doc/refman/5.7/en/performance-schema.html) for more information. If you enable the performance schema, the performance will be affected.
```
Comment (queenypingcap) [inline op-guide/configuration.md]: The value can be (true) or (false).

File: op-guide/configuration.md [hunk 4]
```diff
@@ -43,13 +50,14 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
...
 
-### $TIDB_PPROF environment variable
-+ An environment variable that is used to enable or disable the runtime profiling data via the HTTP server. The Address is at client URL + "/debug/pprof/".
-+ If set $TIDB_PPROF to 0, TiDB will disable pprof. Otherwise TiDB will enable pprof.
+### --report-status
++ enable(true) or disable(false) the status report and pprof tool.
++ default: true
++ the value can be (true) or (false). (true) is to enable metrics and pprof. (false) is to disable metrics and pprof.
```
Comment (queenypingcap) [inline op-guide/configuration.md]: The first "the" -> The

**Comment by shenli** [comment]
PTAL @queenypingcap

**Comment by queenypingcap** [comment]
LGTM

File: op-guide/configuration.md [hunk 5]
```diff
@@ -30,7 +35,9 @@ The default TiDB ports are 4000 for client requests and 10080 for status report.
 ### --status
 + the status report port for TiDB server
 + default: "10080"
-+ This is used to get server status.
++ This is used to get server internal data. The data includes [prometheus metrics](https://golang.org/pkg/net/http/pprof/) and [pprof](https://golang.org/pkg/net/http/pprof/).
```
Comment (qiuyesuifeng) [inline op-guide/configuration.md]: Invalid prometheus metrics url?

**Comment by shenli** [comment]
@qiuyesuifeng PTAL

**Comment by qiuyesuifeng** [comment]
LGTM

**Pull request merged**